localization/english/wc_effects_l_english.yml

```diff
@@ -44,4 +44,17 @@
  change_dark_frenzy_effect_global:0 "Gains $VALUE|+0$ [dark_frenzy|E]"
  change_dark_frenzy_effect_global_neg:0 "Loses $VALUE|-0$ [dark_frenzy|E]"
  change_dark_frenzy_effect_global_past:0 "Gained $VALUE|+0$ [dark_frenzy|E]"
- change_dark_frenzy_effect_global_past_neg:0 "Lost $VALUE|-0$ [dark_frenzy|E]"
\ No newline at end of file
+ change_dark_frenzy_effect_global_past_neg:0 "Lost $VALUE|-0$ [dark_frenzy|E]"
+ 
+ end_scourge_tooltip_global:0 "Ends the [scourge|E]"
+ end_scourge_tooltip_global_past:0 "The [scourge|E] is gone"
+ 
+ imprison_scourge_jailor_consequences_tooltip_first:0 "@warning_icon!#X Other [scourge_jailors_short|E] will punish you#!"
+ imprison_scourge_jailor_consequences_tooltip_first_past:0 "@warning_icon!#X Other [scourge_jailors_short|E] punished you#!"
+ imprison_scourge_jailor_consequences_tooltip_third:0 "@warning_icon!#X Other [scourge_jailors_short|E] will punish [CHARACTER.GetShortUIName|U]#!"
+ imprison_scourge_jailor_consequences_tooltip_third_past:0 "@warning_icon!#X Other [scourge_jailors_short|E] punished [CHARACTER.GetShortUIName|U]#!"
+ imprison_scourge_jailor_consequences_tooltip_global:0 "@warning_icon!#X Other [scourge_jailors_short|E] will take revenge#!"
+ imprison_scourge_jailor_consequences_tooltip_global_past:0 "@warning_icon!#X Other [scourge_jailors_short|E] took revenge#!"
```

Comment:
this game concept alias was deleted but not replaced

![image](https://user-images.githubusercontent.com/46576860/141649931-94e70845-7ba2-4e2b-9b8b-ada1c0567ab5.png)

Reply:
Fixed in 01fd9843fd451379cc4c9ed72a0678ba9de49745.